eoscostarica/eos-rate
# feat(FE): add ratings_cntr column

### GH Show Total Rate Number

#311 

### Steps to test

1. go to EOS RATE webapp
1. go to BPs list page
1. choose one BP and see *RATE* option

### Checklist
